operator_ui/src/components/Icons/Success.js

```diff
@@ -16,7 +16,7 @@ const Success = props => (
     </defs>
     <title>{'Artboard 1'}</title>
     <circle cx={24} cy={24} r={22} fill="none" stroke="#fff" strokeWidth={4} />
-    <circle data-name="path-1" cx={24} cy={24} r={20} fill="#e8faf1" />
+    <circle cx={24} cy={24} r={22} fill="none" stroke={props.isForStatusCard ? '#17c671' : '#fff'} strokeWidth={4} />
```

Comment:
@rupurt I don't think you're going to love what I've done here and https://github.com/smartcontractkit/chainlink/pull/1392/files#diff-248756b17013f8d03720fdf37721d349R101